src/pages/patientView/patientHeader/SignificantMutationalSignatures.tsx

```diff
@@ -0,0 +1,143 @@
+export const MIN_NUMBER_OF_MUTATIONS_STATEMENT = "There are not enough mutations to call mutational signatures for this case.";
+export const DUBIOUS_NUMBER_OF_MUTATIONS_THRESHOLD = 15;
+export const DUBIOUS_NUMBER_OF_MUTATIONS_STATEMENT = "This case has a low number of mutations, signatures are highly uncertain.";
+
+interface ISignificantMutationalSignaturesProps {
+    data: IMutationalSignature[];
+    metadata: IMutationalSignatureMeta[];
+    uniqueSampleKey: string;
+}
+
+export default class SignificantMutationalSignatures extends React.Component<ISignificantMutationalSignaturesProps, {}> {
+    public render() {
+        return (
+            <DefaultTooltip
+                placement='bottomLeft'
+                trigger={['hover', 'focus']}
+                overlay={this.tooltipContent(this.props.data, this.props.metadata, this.props.uniqueSampleKey)}
+                destroyTooltipOnHide={false}
+                onPopupAlign={placeArrowBottomLeft}
+            >
+                <span style={{paddingLeft:2}}>
+                    <a>(Significant Mutational Signatures)</a>
+                </span>
+            </DefaultTooltip>
+        );
+    }
+
+    private tooltipContent(mutationalSignatureData: IMutationalSignature[], mutationalSignatureMetaData: IMutationalSignatureMeta[], uniqueSampleKey: string) {
+
+        const significantMutationalSignaturesForSample: ISignificantMutationalSignaturesForSample = prepareMutationalSignaturesForHeader(mutationalSignatureData, mutationalSignatureMetaData, uniqueSampleKey);
+        let hasMutationalSignatures: boolean = true;
+
+        if(_.isEmpty(significantMutationalSignaturesForSample.significantSignatures)){
+            hasMutationalSignatures = false;
+        }
+
+        return (
+            <div style={{maxWidth:250}}>
+                {/*<h6 style={{textAlign:"center"}}>*/}
+                    {/*Mutational Signature Confidences*/}
+                {/*</h6>*/}
+                <span className="small">
+                    {significantMutationalSignaturesForSample.confidenceStatement}
+                </span>
+                <hr style={{marginTop:10,marginBottom:10}}/>
+                {hasMutationalSignatures &&
+                    <table>
+                    <th>Significant Mutational Signatures</th>
+                        <th>Exposure</th>
+                    {Object.keys(significantMutationalSignaturesForSample.significantSignatures).map((significantSignatureValue)=>
+                        <tr><td style={{paddingTop: 3}}>{significantSignatureValue}</td>
+                        <td style={{paddingTop: 3}}>{this.progressBar(significantMutationalSignaturesForSample.significantSignatures[significantSignatureValue].toString(), false)}</td>
+                        </tr>)}
+                    </table>
+                }
+
+            </div>
+        );
+    }
+
+    public progressBar(confidence:string, determineColor:boolean) {
```

Comment:
this doesnt need anything in the class right? like it doesn't reference `this`. so I dont think it needs to be an instance method. you can move it out as a helper method, maybe into a file called `MutationalSignaturesUtils.tsx`